src/components/Icon/icons.enum.ts

```diff
@@ -6,6 +6,8 @@ import assignmentSignatureOutlinedUrl from "./svg/assignment_signature-outlined.
 import assignmentTurnedFilledUrl from "./svg/assignment_turned-filled.svg";
 import barChartFilledUrl from "./svg/bar_chart-filled.svg";
 import checkOutlinedUrl from "./svg/check-outlined.svg";
+import checkRectangleEnabledFilledUrl from "./svg/check_rectangle_enabled-filled.svg";
+import checkRectangleMinusFilledUrl from "./svg/check_rectangle_minus-filled.svg";
```

Comment:
Mb better name it check mark and minus or like that, because it just icon it can have not one business logic.